Form-Controls/styles.css

```diff
@@ -0,0 +1,47 @@
+
+h1{
+    text-align: center;
+    color: #354f52;
+}
+
+h2{ 
+    text-align: center;
+    color: #354f52;
+    font-size: 12px;
+}
+
+div{
+    margin-bottom: 20px;
+}
+
+.product-form{
+    background-color: #588157;
+    margin: 0px 10em;
```

Comment:
How does em sizing function? What would happen if a user was using this page at 200% text size?